docs/knowledgebase/getting-started/index.md

```diff
@@ -3,24 +3,19 @@ title: Installation
 ---
 
 This page will guide you through the steps needed to prepare a computer for Substrate development.
-Since Substrate is built with [the Rust programming language](https://www.rust-lang.org/), the first
-thing you will need to do is prepare the computer for Rust development - these steps will vary based
-on the computer's operating system. Then, you can use the helpful utilities from the Rust toolchain
-to configure the Rust development environment - these steps will be the same for all the
-(Unix-based) operating systems discussed on this page.
-
-## Operating System-Dependent Set-Up
-
-Follow the steps for your computer's operating system before installing and configuring the Rust
-toolchain.
+As Substrate is built with [the Rust programming language](https://www.rust-lang.org/), the first
+thing you will need to do is prepare the development enviroment, these steps will vary based
+on the computer's operating system. You can utilize helpful utilities from the Rust toolchain
+to configure the Rust development environment. Note that these steps will be the same for all 
+Unix-based operating systems discussed on this page.
 
 ### Unix-Based Operating Systems
 
-Substrate development is easiest on Unix-based operating systems, like macOS or Linux. The examples
+Substrate development is optimized for Unix-based operating systems like MacOS or Linux. The examples
 in the Substrate [Tutorials](../../../../tutorials) and [Recipes](https://substrate.dev/recipes/)
-use Unix-style terminals to demonstrate how to interact with Substrate from the command line.
+use the terminal to demonstrate how to interact with Substrate from the command line.
 
-#### macOS
+#### MacOS
```

Comment:
macOS with a small **m** is actually correct. See: https://en.wikipedia.org/wiki/MacOS#macOS

Reply:
thanks, please revert